scenario.py

```diff
@@ -60,11 +60,19 @@ def __init__(self, params, experiment_path):
 
         self.federated_test_score = int
 
+        # Define how federated learning aggregation steps are weighted. Toggle between 'uniform' and 'data-volume'
+        # Default is 'uniform'
+        if "aggregation_weighting" in params:
+            self.aggregation_weighting = params["aggregation_weighting"]
+        else:
+            self.aggregation_weighting = "uniform"
+
         self.node_list = []
 
         self.contributivity_list = []
 
         self.epoch_count = 40
+        self.minibatch_count = 20
```

Comment:
Specify value in config file so we can easily launch experiments with different values of minibatch.

Reply:
Ok, will do

Reply:
Ok, done